pkg/tsdb/loki/streaming.go

```diff
@@ -0,0 +1,201 @@
+package loki
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"time"
+
+	"cuelang.org/go/pkg/strings"
```

Comment:
should this just be the stdlib `"strings"` package?

Reply:
:facepalm: